f5_openstack_agent/lbaasv2/drivers/bigip/network_helper.py

```diff
@@ -548,7 +606,7 @@ def delete_fdb_entry(
                                 partition=partition)
 
         records = self.get_fdb_entry(
-            bigip, tunnel_name, mac_address, partition=partition)
```

Comment:
Generally, need to look for places where there is a named param in the function def and we are not explicitly citing that in the function call.